example-config.yml

```diff
@@ -349,6 +349,8 @@ itinerary:
   # This flag disables the dot between legs in the metro itnerary summary, replacing it with
   # a gray background
   disableMetroSeperatorDot: false
+  # Show the number of minutes of delay or early beneath a time in the itinerary body
+  showLateness: true
```

Comment:
nitpicky but i think this should be something like `showScheduleDeviation` (Not married to this) since setting this to false would remove late or early.